tests/test_fields.py

```diff
@@ -118,6 +127,19 @@ def test_remove_with_empty_string(self):
         self.assertEqual(serializer.validated_data['created'], uploaded_image.created)
         self.assertIsNone(serializer.validated_data['file'])
 
+    def test_fallback_to_pil_if_not_detected_by_imghdr(self):
+        """
+        Passing a sample image which goes undetected by imghdr should
+        still be detected by PIL.
+        """
+        now = datetime.datetime.now()
+        file = UNDETECTABLE_BY_IMGHDR_SAMPLE
+        uploaded_image = UploadedBase64Image(file=file, created=now)
+        serializer = UploadedBase64ImageSerializer(instance=uploaded_image, data={'created': now, 'file': ''})
```

Comment:
`file` is not send to the serializer. It would be better if we provide the file to the serializer and check if the returned file's content type is `image/jpeg`.

We could also add something like `self.assertIsNone(imghdr.what("test.jpeg", base64.b64decode(file)))`. to show that imghdr can not detect the mime type of the given image.

Reply:
Sorry my bad I mixed up test cases. I have pushed an update please have a look.